src/api/procedures/reclaimCapitalDistribution.ts

```diff
@@ -0,0 +1,80 @@
+import { DividendDistribution, PolymeshError, Procedure, SecurityToken } from '~/internal';
```

Comment:
Please rename the procedure, file and test file to `reclaimDividendDistributionFunds`